### ISSUE TITLE
travis: CI tests failing due to font unavailablility

### ISSUE THREAD

**Issue by kortschak** [gonum/plot#293]
Our tests are failing with many log lines like:

```
2016/07/09 05:24:04 open ../resource/font/Liberationrr.ttf: no such file or directory
2016/07/09 05:24:04 open ../resource/font/luxisr.ttf: no such file or directory
2016/07/09 05:24:04 No font set, and no default font available.
```

Has something changed on Travis that would explain this?

/cc @sbinet @jonlawlor

**Comment by kortschak** [comment]
This is due to https://github.com/llgcode/draw2d/issues/94. The look up path in that code is relative to the cwd, which is not `$GOPATH/src/github.com/llgcode/draw2d/foo`.

**Comment by kortschak** [comment]
Fix PR send to llgcode/draw2d, see #105.

**Comment by sbinet** [comment]
can this be closed? (as llgcode/draw2d#106 has been merged upstream)

**Comment by kortschak** [comment]
Thanks.
